cfme/tests/containers/test_reports.py

```diff
@@ -0,0 +1,198 @@
+# -*- coding: utf-8 -*-
+import re
+
+import pytest
+
+from cfme.containers.provider import ContainersProvider
+from cfme.intelligence.reports.reports import CannedSavedReport
+from utils import testgen
+from utils.blockers import BZ
+
+
+pytestmark = [
+    pytest.mark.usefixtures('setup_provider'),
+    pytest.mark.meta(
+        server_roles='+ems_metrics_coordinator +ems_metrics_collector +ems_metrics_processor'),
+    pytest.mark.tier(1)]
+pytest_generate_tests = testgen.generate([ContainersProvider], scope='function')
+
+
+@pytest.fixture(scope='module')
+def node_hardwares_db_data(appliance):
+
+    """Grabbing hardwares table data for nodes"""
+
+    db = appliance.db
+    hardwares_table = db['hardwares']
+    container_nodes = db['container_nodes']
+
+    out = {}
+    for node in db.session.query(container_nodes).all():
+
+        out[node.name] = hardwares_table.__table__.select().where(
+            hardwares_table.id == node.id
+        ).execute().fetchone()
+
+    return out
+
+
+@pytest.fixture(scope='function')
+def pods_per_ready_status(provider):
```

Comment:
I'll do it later, I just wan't to prevent bureaucracy and add reports testing coverage ASAP, I'm adding a comment for that